public/swagger-ui/css/print.css

```diff
@@ -0,0 +1,1367 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
Looking at the swagger-ui docs, Grunt is not required to get it working, only for developing swagger itself. I did have to sudo to run `npm install` successfully, though.